src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -5,63 +5,115 @@ import moment from 'moment';
 import * as customPropTypes from '../prop-types';
 
 // Helper to derive the download link props.
-const deriveLinkProps = form => {
+const deriveLinkPropsFromFormURL = url => {
   const linkProps = {};
+  if (!url) return linkProps;
 
-  const isSameOrigin = form?.attributes?.url.startsWith(window.location.origin);
-  const isPDF = form?.attributes?.url.toLowerCase().includes('.pdf');
+  const isSameOrigin = url.startsWith(window.location.origin);
+  const isPDF = url.toLowerCase().includes('.pdf');
 
   if (!isSameOrigin || !isPDF) {
     // Just open in a new tab if we'd otherwise hit a CORS issue or if the form URL isn't a PDF.
     linkProps.target = '_blank';
   } else {
     // Use HTML5 `download` attribute.
-    linkProps.download = true;
+    linkProps.download = decodeURI(url.split('/').pop());
```

Comment:
It looks like we do this so that the downloaded file name has only the last part of the URL but I thought that was already the default behavior for when the `download` property is just true?

```
<a href="https://example.com/download.pdf" download>Some PDF</a>  
```

-> Downloads as `download.pdf`

Reply:
Yeah, when a value is left off of the download attribute, the file name is generated from last part of the url.
The real only reason I added the file name is because of the download example on this [comment](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/152#issuecomment-553127340). I did some of my own research on whether or not a value is needed in the download attribute. I didn't find anything that stated the attribute needed a vaule; however, I did go with the example.

Reply:
I think it was provided in that example just for completeness (or maybe we did it have a bug with it at that time 🤷 ) but TBH I think I prefer we leave it off for simplicity in our code.  I like having the browser do that work for us. If we find inconsistency of the download name w/o this though we can include it back. I

Reply:
Yeah, that sounds good to me.